apis/core/v1alpha1/constants.go

```diff
@@ -19,6 +19,9 @@ const (
 	// OperationAnnotation is the annotation that specifies a operation for a component
 	OperationAnnotation = "landscaper.gardener.cloud/operation"
 
+	// TimeoutAnnotation is used to recognize timeouts in deployitems
+	TimeoutAnnotation = "landscaper.gardener.cloud/timeout"
```

Comment:
maybe we not specify the timeout data here but rather the time the reconcile was triggered.
And then compare `currentTime > reconcile-trigger-date`

Reply:
ah I just saw that the actual time is set.
So I would just rename the annotation to reflect what is actually is.
Something like `reconcile-time` or `reconcile-trigger-time`